### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards iOS - FoodOrder-]: 'How would you like your steak prepared?' label is not associated with its Radio button options. (Screen reader)

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7853]
### Target Platforms

iOS

### SDK Version

Version 1.0(2.4.0-beta.22.9.4.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**

Device: iPhone 11

iOS:15.5.1

App Version: Version 1.0(2.4.0-beta.22.9.4.1)

Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.0" control and select it.
4. Navigate on  "V1.0" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "FoodOrder" control and select it.
7. Navigate the 'FoodOrder' screen and navigate to 'Steak' button.
8. Double tap to activate 'Steak' and navigate using swipe gesture and observe the issue.

**Actual Result:**
'How would you like your steak prepared?' label is not associated with radio button options under Steak. Currently it is announcing as 'Left pharen Right pahren comma Rare button'

**Expected Result:**
'How would you like your steak prepared?' label is not associated with radio button options under Steak. It should announce as 'How would you like your steak prepared Rare radio button not selected'.

**User Experience:**
It will impact the screen reader user if the label is not associated with its radio button as user wont be able to understand about the radio button option whether they are selected or not and radio button option comes under which controls.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.03.02

**Observation:**
1. Voice Over also does not announce the information as Radio button and for non selected it is not announce as not selected. 

**Note:**
1. Same issue is repro under 'FoodOrderWithValidation' screen for the 'Steak','Chicken' and 'tofu' option.
2. Same issue is repro with Chicken and tofu options.

### Screenshots

https://user-images.githubusercontent.com/95913748/190175482-f7629bd8-29e3-4627-86ad-029953188f59.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
Please use updated v1.3 FoodOrderWithValidation for this test. the issue is fixed with v1.3 label feature. thanks!

**Comment by vagpt** [comment]
We have verified this issue on environment below.

**Test Environment:**
Device: iPhone 11
iOS Version: 16.2
App Version: 168
